src/aura/accountsApp/accountsApp.app

```diff
@@ -0,0 +1,4 @@
+<aura:application extends="force:slds">
+    <c:cmp1 />
```

Comment:
Naming it's important, we must get better at  this, the idea of a comp name is that I can get valuable info about it, cmp1 does not tell me anything about what that comp does